### ISSUE TITLE
Nesting with namespace does not work when passed to i18n.t()

### ISSUE THREAD

**Issue by rally25rs** [i18next/i18next#1474]
## 🐛 Bug Report

Nesting that includes a namespace no longer works when passed to `t()`.

## To Reproduce

Sample: https://codesandbox.io/s/mystifying-nightingale-hc8o2?file=/src/index.js

## Expected behavior

Given the translations
```js
        translation: {
          key: "hello world",
          hasNesting: "$t(translation:key)"
        }
```

All of the following should produce the same result `hello world`
```js
i18n.t('hasNesting'); // 'hello world' (correct)
i18n.t('key'); // 'hello world' (correct)
i18n.t('$t(key)'); // 'hello world' (correct)
i18n.t('translation:key'); // 'hello world' (correct)

i18n.t('$t(translation:key)'); // 'key)' (incorrect)
```

## Your Environment

- *runtime version*: firefox
- *i18next version*: 19.5.1
- *os*: Mac
- *any other relevant information*

**Comment by adrai** [comment]
Fixed with v19.5.2

**Comment by adrai** [comment]
If you like this module don’t forget to star this repo. Make a tweet, share the word or have a look at our https://locize.com to support the devs of this project.

There are many ways to help this project 🙏
